modules/end-user-guide/partials/proc_listing-workspaces.adoc

```diff
@@ -1,15 +1,57 @@
 
 [id="listing-workspaces"]
-= Listing workspaces
+= Listing all workspaces
 
-Explain why `--all-namespaces`.
+You can list your workspaces on a command line.
 
+.Prerequisites
+
+* An active `{orch-cli}` session on the cluster. See {orch-cli-link}.
```

Comment:
To match prereqs in "creating devworkspaces", perhaps we should include that you also need `get` and `list` permissions for DevWorkspaces in the relevant namespace.

Reply:
Thank you, fixed